utilities/test_suite/HOST/Tensor_misc_host.cpp

```diff
@@ -178,7 +200,11 @@ int main(int argc, char **argv)
                     fill_mean_stddev_values(nDim, maxSize, meanTensor, stdDevTensor, qaMode, axisMask, scriptPath);
 
                 startWallTime = omp_get_wtime();
-                rppt_normalize_host(inputF32, srcDescriptorPtrND, outputF32, dstDescriptorPtrND, axisMask, meanTensor, stdDevTensor, computeMeanStddev, scale, shift, roiTensor, handle);
+
```

Comment:
Why do we have the function for u8 in misc tests? you have already added the same function in the image unit tests

Reply:
I removed U8 call in misc test suite .